pkg/virt-handler/migration.go

```diff
@@ -22,17 +22,32 @@ package virthandler
 import (
 	"fmt"
 	"os"
+	"time"
+
+	"kubevirt.io/client-go/log"
 
 	"gopkg.in/yaml.v2"
 )
 
 // FindMigrationIP looks for dedicated migration network migration0 using the downward API and, if found, sets migration IP to it
 func FindMigrationIP(networkStatusPath string, migrationIp string) (string, error) {
 	var networkStatus []NetworkStatus
+	var err error
+	var dat []byte
 
-	dat, err := os.ReadFile(networkStatusPath)
-	if err != nil {
-		return "", fmt.Errorf("failed to read network status from downwards API")
+	for i := 0; i < 10; i++ {
+		dat, err = os.ReadFile(networkStatusPath)
+		if err != nil {
+			return "", fmt.Errorf("failed to read network status from downwards API")
+		}
+		if len(dat) != 0 {
+			break
+		}
+		log.Log.Warningf("network status still empty after %d seconds, retrying.", i)
+		time.Sleep(time.Second)
+	}
+	if len(dat) == 0 {
+		return "", fmt.Errorf("downward API network status empty")
```

Comment:
Do you think it might be worth it to introduce an exponential backoff? WDYT?

Reply:
Thanks for the review @fossedihelm 
This PR doesn't work, as the annotation is only present when some network component is installed, so that won't work.
I need to rework it, I think I'll just call `ip addr` instead of relying on some annotation.